### ISSUE TITLE
[Feature]: [UI/UX] Improve AI Troubleshoot page design and visual hierarchy

### ISSUE THREAD

**Issue by amarakaushal-max** [rishabh0510rishabh/EnvForage#342]
### Description
The AI Troubleshoot page currently feels visually flat and inconsistent. Several small frontend improvements could significantly enhance the user experience without touching any existing functionality.

### Current Issues
- The brain icon has no visual weight — it blends into the white background
- The diagnostic report text area uses a dark grey background that clashes with the otherwise light UI
- The form card has no border or shadow, making it feel ungrounded on the page
- Input fields lack visible focus states, making keyboard navigation feel broken
- The "Analyze" CTA button doesn't stand out as the primary action
- No quick-access options for common target profiles forces users to open the dropdown every time

### Screenshots

<img width="1890" height="951" alt="Image" src="https://github.com/user-attachments/assets/d353523b-10bd-42e1-bf90-10cb75e8bd80" />

### Proposed Changes
- Add a gradient background to the brain icon to give it visual weight
- Fix the text area background to match the light theme (use a consistent surface colour)
- Add a subtle border to the form card so it feels grounded
- Add focus ring styles on all inputs (matching the existing purple/pink brand colours)
- Make the Analyze button full-width with a gradient to clearly signal it as the primary CTA
- Add quick-select pills for common targets (PyTorch + CUDA, TensorFlow, JAX, etc.) above the form
- Use CSS variables throughout for automatic dark mode support

### Additional Notes
All proposed changes are purely frontend (HTML/CSS) and do not touch any existing logic or functionality. I am willing to work on this if it gets approved and assigned.

**Comment by amarakaushal-max** [comment]
Hi @rishabh0510rishabh 👋
I would love to work on this issue as part of GSSoC 2026! I have already analyzed the current UI and have a clear plan for the improvements mentioned above. I can get started right away.
Could I please be assigned this issue?

**Comment by rishabh0510rishabh** [comment]
assigned @amarakaushal-max , if you need any help feel free to reach out
